### ISSUE TITLE
Incorrect 'unbalanced parenthesis' for url() with no quotes.

### ISSUE THREAD

**Issue by cjwelborn** [Kronuz/pyScss#350]
If a file contains a url like this:
```sass
.logo {
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAAGCAYAAADgzO9IAAAAP0lEQVQIHWWMAQoAIAgDR/QJ/Ub//04+w7ZICBwcOg5FZi5iBB82AGzixEglJrd4TVK5XUJpskSTEvpdFzX9AB2pGziSQcvAAAAAAElFTkSuQmCC);
}
```

An exception is raised with 'unbalanced parenthesis', possibly because the url contains a `;`. Adding quotes around the url fixes the issue, but the quotes are supposed to be optional [as far as I can tell](http://www.w3.org/TR/CSS2/syndata.html#value-def-uri).

**Comment by eevee** [comment]
Fixed in 1.3.5, thanks!

**Comment by cjwelborn** [comment]
Awesome, thanks for the fix.
